src/community/china/_os-settings.md

```diff
@@ -0,0 +1,106 @@
+{% assign id =  include.os | downcase -%}
+
+{% if id == 'windows' -%}
+{% assign shell = 'Powershell' -%}
+{% assign prompt = 'C:\>' -%}
+{% assign envvarset = 'C:\> $env:' -%}
+{% assign setpath = 'C:\> $env:PATH = $pwd.PATH + "/flutter/bin",$env:PATH -join ";"' -%}
+{% assign comtoset = '$env:' -%}
+{% assign unzip='Extract-Archive' %}
+{% capture permaddexample -%}
+$newPath = $pwd.PATH + "/flutter/bin",$env:PATH -join ";"
+[System.Environment]::SetEnvironmentVariable('Path',$newPath,User)
+[System.Environment]::SetEnvironmentVariable('PUB_HOSTED_URL','https://pub.flutter-io.cn',User)
+[System.Environment]::SetEnvironmentVariable('FLUTTER_STORAGE_BASE_URL','https://storage.flutter-io.cn',User)
+{% endcapture -%}
+{% else -%}
+{% assign shell = 'your terminal' -%}
+{% assign prompt = '$' -%}
+{% assign envvarset = '$ export ' -%}
+{% assign setpath = '$ export PATH="$PWD/flutter/bin:$PATH"' -%}
+{% assign comtoset = 'export' -%}
+{% assign unzip='unzip' %}
+{% capture permaddexample -%}
+cat <<EOT >> ~/.zprofile
+{{envvarset}}PUB_HOSTED_URL="https://pub.flutter-io.cn"
+{{envvarset}}FLUTTER_STORAGE_BASE_URL="https://storage.flutter-io.cn"
+{{setpath}}
+EOT
+{% endcapture -%}
+{% endif -%}
+
+<div id="{{id}}" class="tab-pane
+  {%- if id == 'windows' %} active {% endif %}"
+  role="tabpanel" aria-labelledby="{{id}}-tab" markdown="1">
+
+1. Open a new Terminal window to prepare running scripts.
+
+1. Set `PUB_HOSTED_URL` to your mirror site.
+   In {{shell}}, run the following command:
+
+   ```terminal
+   {{envvarset}}PUB_HOSTED_URL="https://pub.flutter-io.cn"
+   ```
+
+1. Set `FLUTTER_STORAGE_BASE_URL` to your mirror site.
+   In {{shell}}, run the following command:
+
+   ```terminal
+   {{envvarset}}FLUTTER_STORAGE_BASE_URL="https://storage.flutter-io.cn"
+   ```
+
+1. Download the Flutter archive from your mirror site.
+   In your preferred browser, go to
+   [Flutter SDK archive](https://flutter.cn/docs/release/archive?tab={{id}}).
```

Comment:
These instruction will be incorrect for most other mirror sites as they don't host a web version of our archive like CFUG does.

Perhaps you can provide two paths. One for the CFUG mirror and link to their archive, "Otherwise, follow the instructions in [Download Flutter archives based on a mirrored URL](#download-flutter-archives-based-on-a-mirrored-url)" or similar.

Reply:
This entire procedure starts with "All examples that follow presume that you are using the CFUG mirror." Keeping as-is.